bin/rake

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env ruby
```

Comment:
I think we might need this after all; looked it up and the `#!` thing is a "shebang" and actually executes stuff. https://en.wikipedia.org/wiki/Shebang_(Unix)